01_Introduction_to_Citsci_Pipeline.ipynb

```diff
@@ -402,7 +669,7 @@
     "print('Send the data to Zooniverse')\n",
     "subject_set_name = \"\"\n",
     "if len(subject_set_name) != 0:\n",
-    "    cit_sci_pipeline.send_image_data(subject_set_name, batch_dir, manifest)\n",
```

Comment:
This might be due to an outdated version of the `cit_sci_pipeline`. Can you run `pip install --update rubin.citsci`? I believe the version should be 0.3.2.